packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -114,7 +114,7 @@ const builtInFlags = [
     // Complex configs
     {
         name: 'env',
-        usage: '--env',
+        usage: '--env <variable> | --env <variable> --env <variable=value>',
```

Comment:
Ah I think we should also mention that we support nested flags here, var1.var2.var3=value, will PR

/cc @alexander-akait